content/chainguard/chainguard-images/reference/apko/overview.md

```diff
@@ -1,7 +1,7 @@
 ---
-title: "Image Overview: apko"
+title: "Image Overview: Apko"
 type: "article"
-description: "Overview: apko Chainguard Images"
+description: "Overview: Apko Chainguard Image"
```

Comment:
How did the `a` in `apko` get title cased?

Reply:
Ah! I used a function to title case all image names when generating the pages, forgot about apko and melange specific cases. Should I revert all?

About mobile: ugh! yeah that data is hard to format to look nice on mobile because the digests are so long. I will need some help to figure this one out.

Reply:
Fixed this!